lib/librom.i

```diff
@@ -0,0 +1,93 @@
+%module librom
+
+//TODO: Untie this from LC.
+%include /usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/include/mpi4py/mpi4py.i
+%mpi4py_typemap(Comm, MPI_Comm);
+%include <std_string.i>
+%include <std_vector.i>
+// Instantiate templates used by example
+namespace std {
+   %template(IntVector) vector<int>;
+   %template(DoubleVector) vector<double>;
+}
+%include <std_shared_ptr.i>
+%include <cpointer.i>
+
+/* Create some functions for working with "int *" */
+%pointer_functions(int, intp);
+
+/* Create some functions for working with "double *" */
+%pointer_functions(double, doublep);
+
+%{
+    #define SWIG_FILE_WITH_INIT
+    #include "utils/CSVDatabase.h"
+    #include "utils/HDFDatabase.h"
+    #include "linalg/Vector.h"
```

Comment:
Some files have not been included as I have not been able to get them to work yet, such as the greedy algorithm files.

Reply:
DifferentialEvolution must also be eventually added, but it's not a big deal for now.